backend/tests/integration/test_ttl_cache_behavior.py

```
@@ -0,0 +1,249 @@
+
+    def test_cache_performance_characteristics(self, mock_settings, mock_dependencies):
+        """Test performance characteristics of cache operations."""
+        service = AlertService(mock_settings)
+        
+        # Generate test data
+        alert_ids = [str(uuid.uuid4()) for _ in range(100)]
+        session_ids = [f"session-{i}" for i in range(100)]
+        
+        # Measure insertion time
+        start_time = time.time()
+        for alert_id, session_id in zip(alert_ids, session_ids):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)